spinalcordtoolbox/scripts/sct_analyze_texture.py

```diff
@@ -206,7 +206,12 @@ def mean_angle(self):
             # Average across angles and save it as wrk_folder/fnameIn_feature_distance_mean.extension
             fname_out = im_m + str(self.param_glcm.distance) + '_mean' + extension
             run_proc('sct_image -i ' + ' '.join(im2mean_lst) + ' -concat t -o ' + fname_out)
-            run_proc('sct_maths -i ' + fname_out + ' -mean t -o ' + fname_out)
+
+            img = Image(fname_out)
+            dim_idx = 3 # 3 dimensions + time
```

Comment:
Nit: I don't think the comment here is quite accurate...

```suggestion
            dim_idx = 3  # img is [x, y, z, angle] so specify 4th dimension (angle)
```